src/System.Security.Cryptography.Csp/tests/RSACryptoServiceProviderBackCompat.cs

```diff
@@ -116,6 +122,63 @@ public static void SignHash_PublicKey()
             }
         }
 
+        [Fact]
+        public static void OldVerifyHash()
+        {
+            byte[] dataHash, signature;
+
+            using (HashAlgorithm hash = SHA256.Create())
+            {
+                dataHash = hash.ComputeHash(TestData.HelloBytes);
+            }
+
+            // Use the new RSA SignHash method to generate the signature
+            using (RSA rsa = RSAFactory.Create())
+            {
+                rsa.ImportParameters(TestData.RSA2048Params);
+                signature = rsa.SignHash(dataHash, new HashAlgorithmName("SHA256"), RSASignaturePadding.Pkcs1);
+            }
+
+            // Verify using the old RSACryptoServiceProvider VerifyHash
+            using (var rsa = new RSACryptoServiceProvider())
+            {
+                rsa.ImportParameters(s_publicOnly);
+                Assert.True(rsa.VerifyHash(dataHash, "SHA256", signature));
```

Comment:
This is more complicated than I expected, but it's good.  I was just looking for something like `ApiInterop_NewToOld` :smile:.

Since this verifies that VerifyHash works with a public-only object it's technically better.  I'm happy either with this how it is, or compressed down.